rss_pipeline/transform_rss.py

```diff
@@ -208,9 +209,16 @@ def transform_daily_mail_xml_file(daily_mail_xml_file: str, sentiment_analyser:
 
     vader = SentimentIntensityAnalyzer(lexicon_file="vader_lexicon.txt")
 
-    bbc_articles_df = transform_bbc_xml_file(BBC_UK_NEWS_XML_FILE_NAME, vader)
-    daily_mail_articles_df = transform_daily_mail_xml_file(
-        DAILY_MAIL_UK_NEWS_XML_FILE_NAME, vader)
+    # Multithreading to make the tasks run faster
+    with ThreadPoolExecutor(max_workers=2) as executor:
+        future_bbc = executor.submit(
+            transform_bbc_xml_file, BBC_UK_NEWS_XML_FILE_NAME, vader)
+        future_daily_mail = executor.submit(
+            transform_daily_mail_xml_file, DAILY_MAIL_UK_NEWS_XML_FILE_NAME, vader)
+
+        # Wait for both tasks to complete
+        bbc_articles_df = future_bbc.result()
+        daily_mail_articles_df = future_daily_mail.result()
```

Comment:
Has this been tested on the Lambda via the ECR?

Reply:
this is used when running transform locally, not used elsewhere. Feel like it will cause more issues that aren't worth the slightly faster processing time

Reply:
Sounds good 👍